terraform/modules/benchmark/service/Gemfile

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
```

Comment:
We either need to use Gemfile to install gems or manually install them via `gem install ..`. Is there a point in having those dependencies in both places? The gemfile is the preferred way.